docs/timeline/index.html

```diff
@@ -1571,7 +1577,7 @@ <h2 id="Editing_Items">Editing Items</h2>
 };</pre>
 
   <p>
-    Editing can be enabled/disabled for specific items. Setting the property <code>editable</code> to <code>true</code> or <code>false</code> on a data item will override the timeline option.
+    Editing can be enabled/disabled for specific items. Setting the property <code>editable</code> to <code>true</code> or <code>false</code> on a data item will override the timeline option depending on the value of <code>timeline.editable.overrideItems</code>.
```

Comment:
> ...will override the timeline option depending on the value of <code>timeline.editable.overrideItems</code>.

I would propose:
...will override the timeline option except <code>timeline.editable.overrideItems</code> is set to <code>true</code>.